### PR TITLE
Feature/ready for recording

### PR DESCRIPTION
**Pull request by promix17**

### CODE REVIEW

File: spec/support/fast_helper.rb [hunk 1]
```diff
@@ -44,21 +45,35 @@ def wait_test_run_for_finish
...
     when 'passed'
-      puts 'FAST tests are passed'.green
+      puts "FAST tests have passed #{link}".green
     end
   end
 
   private
 
+  def frontend_url
+    default = 'https://us1.my.wallarm.com'
```
Comment (d0nt-panic) [inline spec/support/fast_helper.rb]: May be we could move this to constant?

File: spec/support/fast_helper.rb [hunk 2]
```diff
@@ -44,21 +45,35 @@ def wait_test_run_for_finish
...
+
+    url = "https://#{ENV['WALLARM_API_HOST']}/v1/user"
+
+    response = RestClient.post url, {}, auth_headers
+
+    JSON.parse(response.body)['body']['frontend_url'] || default
+  rescue
```
Comment (d0nt-panic) [inline spec/support/fast_helper.rb]: `rescue StandardError`

**Review by d0nt-panic — COMMENTED** [review]
I have these comments, but they are not necessary

File: spec/support/fast_helper.rb (hunk 1 above)
Comment (promix17) [reply spec/support/fast_helper.rb]: Done, removed after api deploy

File: spec/support/fast_helper.rb (hunk 2 above)
Comment (promix17) [reply spec/support/fast_helper.rb]: Done, removed after api deploy

**Pull request closed without merging**